### PR TITLE
Add plot ridge

### PR DESCRIPTION
**Pull request by mihagazvoda**
Hi,
I love the package!
I added `plot_ridges` function and fixed packages imports and depends so there's no need for additional package imports.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,12 +51,10 @@ install.packages(c("devtools", "mapproj", "tidyverse", "gtools"))
 devtools::install_github("marcusvolz/strava")
 ```
 
-### Load the libraries
+### Load the librariy
```
Comment (hugovk) [inline README.md:54]: Little typo
```suggestion
### Load the library
```

**Comment by marcusvolz** [comment]
Thanks Miha and Hugo!

**Comment by hugovk** [comment]
Would also be great to see some examples in the README!

**Comment by mihagazvoda** [comment]
Agree! An example would be great because it can also be used for unit
tests. I'm not sure where to get the data (don't want to use my personal).
Any ideas?
I also plan to give a shot to `add_calendar` function and maybe refactor
others.

**Pull request merged**